Apps/Immunization/test/unit/Services.Test/VaccineStatusServiceTests.cs

```diff
@@ -40,21 +45,26 @@ namespace HealthGateway.Immunization.Test.Services
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Major Code Smell", "S125:Sections of code should not be commented out", Justification = "Ignore broken tests")]
```

Comment:
This line should be removed now that we are not disabling any tests by commenting them out.